filetests/isa/x86/legalize-simd.clif

```diff
@@ -1,11 +1,21 @@
 test compile
+set probestack_enabled=false
 target x86_64
 
-function %test_simd() -> i32x4 {
+; use baldrdash calling convention here for simplicity (avoids prologue, epilogue)
+function %test_simd() -> i32x4 baldrdash {
 ebb0:
     v0 = iconst.i32 42
     v1 = splat.i32x4 v0
-    ; sameln: v2 = bitcast.i32x4 v0
-    ; nextln: v1 = x86_pshuf.i32x4 v2
```

Comment:
I realized what I was doing here didn't really match the filecheck documentation (see https://docs.rs/filecheck/0.4.0/filecheck). What this said is: starting at the first line, find something like `v2 = bitcast.i32x4 v0`. What I needed to do is either start checking the first line of the output (e.g. `function ...`) or to start with a `check` directive (e.g. `check: ebb0`) to ground the matching and then use `sameln`/`nextln`.

Reply:
TIL!